core/src/main/java/io/grpc/ServerServiceDefinition.java

```diff
@@ -70,7 +70,7 @@ public String getName() {
    *
    * @param name the fully qualified name without leading slash. E.g., "com.foo.Foo/Bar"
    */
-  ServerMethodDefinition<?, ?> getMethod(String name) {
+  public ServerMethodDefinition<?, ?> getMethod(String name) {
```

Comment:
If the public stays (because HandlerRegistry stays internal), make it `@Internal`.